sgqlc/types/__init__.py

```diff
@@ -593,9 +593,11 @@
 __all__ = (
     'Schema', 'Scalar', 'Enum', 'Union', 'Variable', 'Arg', 'ArgDict',
     'Field', 'Type', 'Interface', 'Input', 'Int', 'Float', 'String',
-    'Boolean', 'ID', 'non_null', 'list_of',
+    'Boolean', 'ID', 'non_null', 'list_of', 'CONVERT_PYNAMES'
 )
 
+CONVERT_PYNAMES = True
```

Comment:
I'd avoid doing this, it would be better to define it in the `schema`, either the global or user-defined schema and then you can query it when using schema-bound types.